packages/mantine/package.json

```
@@ -0,0 +1,97 @@
+{
+  "name": "@rjsf/mantine",
+  "version": "6.0.0-alpha.0",
+  "main": "dist/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "description": "Mantine theme, fields and widgets for react-jsonschema-form",
+  "files": [
+    "dist",
+    "lib",
+    "src"
+  ],
+  "engineStrict": false,
+  "engines": {
+    "node": ">=14"
+  },
+  "scripts": {
+    "build:ts": "tsc -b",
```

You will want to copy the `tsconfig.build.json` file from one of the other themes, like `chakra-ui`
```suggestion
    "build:ts": "tsc -b && tsc-alias -p tsconfig.build.json",
```